app/src/androidTest/java/ch/epfl/sweng/swenggolf/ShowOfferActivityTest.java

```diff
@@ -114,4 +115,9 @@ private boolean isGoogleMapsInstalled() {
             return false;
         }
     }
+
+    @Test
+    public void imageDisplayOnClick() {
+        onView(withId(R.id.show_offer_picture)).perform(scrollTo(), click());
```

Comment:
You aren't checking anything. Isn't it possible to check that the dialog is displayed ?

Reply:
This is false I'm checking that no error is launched

Reply:
Okay but is it possible to test what I suggested ?